mw-webapp/src/utils/arrayToHashMap.ts

```diff
@@ -1,21 +1,27 @@
 /**
- * Uuid props
+ * ArrayToHashMapArgs
  */
-interface UuidProps {
+interface ArrayToHashMapArgs<Entity extends object> {
+
+  /**keyField
+   */
+  keyField: keyof Entity;
 
   /**
-   * Uuid of {@link T}
+   * list
```

Comment:
Mm, list...
How did you add a comment not with a capital letter? Also is it helpful comment?